### PR TITLE
render: remove --config-override-file default and pluralize

### PR DESCRIPTION
**Pull request by mfojtik**
Trimmed version of https://github.com/openshift/cluster-kube-scheduler-operator/pull/17 with just the override changes.

/cc @sttts

### CODE REVIEW

File: cmd/cluster-kube-scheduler-operator/render/render.go [hunk 1]
```diff
@@ -117,10 +116,6 @@ func (r *renderOpts) Validate() error {
 }
 
 func (r *renderOpts) complete() error {
```
Comment (aveshagarwal) [inline cmd/cluster-kube-scheduler-operator/render/render.go]: is this function for future now?

Comment (mfojtik) [reply cmd/cluster-kube-scheduler-operator/render/render.go]: yeah, also complete() is something all commands implement....

**Comment by mfojtik** [comment]
/hold cancel

@aveshagarwal this is now ready

File: cmd/cluster-kube-scheduler-operator/render/render.go [hunk 2]
```diff
@@ -73,10 +74,14 @@ func NewRenderCommand() *cobra.Command {
 	cmd.Flags().StringVar(&renderOpts.assetOutputDir, "asset-output-dir", "", "Output path for rendered manifests.")
 	cmd.Flags().StringVar(&renderOpts.assetInputDir, "asset-input-dir", "", "A path to directory with certificates and secrets.")
 	cmd.Flags().StringVar(&renderOpts.templatesDir, "templates-input-dir", "/usr/share/bootkube/manifests", "A path to a directory with manifest templates.")
-	cmd.Flags().StringVar(&renderOpts.configOverrideFile, "config-override-file", "", "A sparse KubeSchedulerConfiguration."+
-		"componentconfig/v1alpha1 file (default: kube-scheduler-config-overrides.yaml in the asset-input-dir)")
+	cmd.Flags().StringSliceVar(&renderOpts.configOverrideFiles, "config-override-files", nil, "Additional sparse KubeSchedulerConfiguration.componentconfig/v1alpha1 files for customiziation through the installer, merged into the default config in the given order.")
 	cmd.Flags().StringVar(&renderOpts.configOutputFile, "config-output-file", "", "Output path for the KubeSchedulerConfig yaml file.")
 
+	// TODO: Remove these once we break the flag dependency loop in installer
```
Comment (sttts) [inline cmd/cluster-kube-scheduler-operator/render/render.go]: :+1:

**Pull request merged**